lib/commands/deploynodeapp.js

```diff
@@ -72,6 +80,31 @@ var descriptor = defaults.defaultDescriptor({
     name: 'Preserve policies from previous revision',
     shortOption: 'P',
     toggle: true
+  },
+  runtime: {
```

Comment:
I like that better too, just a flag for toggling on the `edge-services` runtime. @AdamMagaluk @jbowen93 what do you think?